free-programming-books.md

```diff
@@ -1059,6 +1059,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Everything you need to know about pointers in C - Peter Hosey](http://boredzo.org/pointers/)
 * [Functional C (1997)](https://research.utwente.nl/files/5128727/book.pdf) - Pieter H. Hartel, Henk Muller (PDF)
 * [Hashing](https://www.smashwords.com/books/view/737188) - Prakash Hegade
+* [Learn C the hard way](https://users.elis.ugent.be/~jrvdnbro/learn_c_the_hard_way.pdf) - Zed A.Shaw (PDF)
```

Comment:
Not an authorized distribution. Please see CONTRIBUTING.md guidelines. please replace with Zed Shaw's authorized web site.